wordpress-mobile/WordPress-Android
# Update to latest gutenberg-mobile

Update to latest gutenberg-mobile.

This should have been done before I merged https://github.com/wordpress-mobile/WordPress-Android/pull/9871 but I jumped to that merge too soon.

To test:

1. Build and run the app
2. Go into a Gutenberg post and verify that the editor loads correctly

Update release notes:
